plugins/engine-configuration/impl/src/main/java/org/pentaho/di/engine/configuration/impl/RunConfigurationManager.java

```diff
@@ -41,10 +40,6 @@ public RunConfigurationManager( List<RunConfigurationProvider> runConfigurationP
     this.runConfigurationProviders = runConfigurationProviders;
   }
 
-  private RunConfigurationManager() {
-    runConfigurationProviders = RunConfigurationProviderFactoryManagerImpl.getInstance().generateProviders();
```

Comment:
We also need to remove this method as well

```RunConfigurationProviderFactoryManagerImpl.getInstance().generateProviders();```

Reply:
fixed